### ISSUE TITLE
Consider exporting argparse error type

### ISSUE THREAD

**Issue by mz8i** [swansonk14/typed-argument-parser#46]
It might be useful to provide some kind of error classes as part of this package.
For example, I use code like this:

```python
from argparse import ArgumentError, ArgumentParser

def file_path(x: str):
   d = Path(x)
   if d.is_dir:
      raise ArgumentError(f"{d} is a directory")
   return d

parser = ArgumentParser()
parser.add_argument('output_file', type=file_path, help="Path to output file")
parser.parse_args()
```

I would like to use TAP for this, so would be nice to import all the classes I need from the single package, rather than still importing error classes from argparse, or writing my own each time.
The easiest would probably be to just expose the classes from `argparse`.

**Comment by martinjm97** [comment]
Hi @mz8i 

Thank for the recommendation! This looks extremely reasonable. I'm a fan. 

Best, 
Jesse

**Comment by swansonk14** [comment]
Hi @mz8i,

We exposed argparse's `ArgumentError` and `ArgumentTypeError` in this commit: ea8254d6a4042b6339e6a41ca0f7cfe772c98848. This functionality is now available in Tap version 1.7.2 as documented in the release notes: https://github.com/swansonk14/typed-argument-parser/releases/tag/v_1.7.2.

Thanks again for the great suggestion!

Best,
Jesse and Kyle
